test/upgrade/prober/prober.go

```diff
@@ -114,11 +116,20 @@ func (p *prober) servingClient() resources.ServingClient {
 	}
 }
 
-func (p *prober) ReportError(t *testing.T, err error) {
-	if p.config.FailOnMissingEvents {
-		t.Error(err)
-	} else {
-		p.log.Warnf("Silenced FAIL: %v", err)
+func (p *prober) ReportErrors(t *testing.T, errors []error) {
+	for _, err := range errors {
+		if p.config.FailOnMissingEvents {
+			t.Error(err)
+		} else {
+			p.log.Warnf("Silenced FAIL: %v", err)
+		}
+	}
+	errsNum := len(errors)
+	if errsNum > 0 && !p.config.FailOnMissingEvents {
```

Comment:
Elsewhere in this file we use `len(errors)` directly in the condition as well as the logging statment, instead of introducing a new variable.  Could we please change this to stay consistent? Thanks!

Reply:
+1 on consistency

Reply:
ok